packages/components/src/components/tooltip/tooltip.test.ts

```diff
@@ -184,50 +186,78 @@ describe('<sd-tooltip>', () => {
 
     // tooltip is hidden again after clicking the button a second time
     expect(body.hidden).to.be.true;
+    expect(document.activeElement).to.not.equal(el);
   });
 
-  it('should ignore click event if focus is triggered first', async () => {
+  it('removes whitespace nodes from the default slot', async () => {
     const el = await fixture<SdTooltip>(html`
-      <sd-tooltip content="This is a tooltip" trigger="click focus">
-        <sd-button>Click or Focus Me</sd-button>
+      <sd-tooltip content="This is a tooltip">
+        <!-- -->
+        <span>Content</span>
+      </sd-tooltip>
+    `);
+
+    const defaultSlot = el.shadowRoot!.querySelector('slot:not([name])')!;
+    const nodes = (defaultSlot as HTMLSlotElement).assignedNodes({ flatten: true }) as HTMLElement[];
+
+    expect(nodes.length).to.equal(1);
+  });
+
+  it('should toggle the tooltip when focus the trigger', async () => {
+    const el = await fixture<SdTooltip>(html`
+      <sd-tooltip content="This is a tooltip" trigger="focus">
+        <sd-button>Click Me</sd-button>
       </sd-tooltip>
     `);
 
     const button = el.querySelector('sd-button')!;
     const body = el.shadowRoot!.querySelector<HTMLElement>('[part~="body"]')!;
 
-    const showSpy = sinon.spy(el, 'show');
-    const hideSpy = sinon.spy(el, 'hide');
+    // tooltip starts hidden
+    expect(body.hidden).to.be.true;
 
+    document.dispatchEvent(new KeyboardEvent('keydown', { key: 'Tab' }));
     button.focus();
-    await waitUntil(() => showSpy.calledOnce);
+    await waitUntil(() => !body.hidden);
 
-    // tooltip is visible after focusing
+    // tooltip is visible after clicking the button
     expect(body.hidden).to.be.false;
 
-    button.click();
-    await waitUntil(() => hideSpy.notCalled);
-
-    // button click is ignored because focus was triggered first
-    expect(body.hidden).to.be.false;
-    expect(showSpy.calledOnce).to.be.true;
-    expect(hideSpy.notCalled).to.be.true;
+    document.dispatchEvent(new KeyboardEvent('keydown', { key: 'Tab' }));
+    button.blur();
+    await waitUntil(() => body.hidden);
 
-    showSpy.restore();
-    hideSpy.restore();
+    // tooltip is hidden again after clicking the button a second time
+    expect(body.hidden).to.be.true;
+    expect(document.activeElement).to.not.equal(el);
   });
 
-  it('removes whitespace nodes from the default slot', async () => {
+  it('should close tooltip when clicking trigger if it was opened by focus', async () => {
```

Comment:
Since the trigger is only `focus`, clicking the button shouldn't hide the tooltip, this is only happening because on L256 you're using `blur()`. I think it would be easier to check the `open` attribute when focusing on the button and then when removing focus.